internal/registry/clientsecretrequest/rest.go

```diff
@@ -70,32 +125,159 @@ func (*REST) Categories() []string {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	t := trace.FromContext(ctx).Nest("create", trace.Field{
-		Key:   "kind",
-		Value: "OIDCClientSecretRequest",
-	})
+	t := trace.FromContext(ctx).Nest("create",
```

Comment:
This API is not going to be called a lot, so its cool to have tracing in here all the time.  The bcrypt calls can take a bit of time, so this may be valuable to know.